gunicorn_config.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+workers = 4
```

Comment:
What happens if we don't complete the scan within the PaaS 10 secs SIGTERM wait?

Can we gracefully fail? i.e. detect it

I guess you'll get a 500 from DocumentmentDownload and retry?

Reply:
As in - when the app is deployed / scaled down? I don't think it's possible for us to handle it here since we won't be able to send the response either way. Document Download would probably have to 503.